src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const jwtService = require('../services/jwtService.js');
+const { ApiError } = require('../exceptions/ApiError.js');
+
+function authMiddleware(req, res, next) {
+  const authHeader = req.headers['authorization'];
+
+  if (!authHeader) {
+    throw ApiError.Unauthorized();
```

Comment:
Use 'next' function to pass error handling to the next middleware in the stack for more centralized error handling. Replace 'throw ApiError.Unauthorized();' with 'next(ApiError.Unauthorized());' 
Do it in all cases